status-loop/getChannels.js

```diff
@@ -1,9 +1,29 @@
 const getRequest = require('../helpers/getRequest')
 const cfg = require('../cfg')
+const db = require('../db')
+
+async function retrieveChannelsFromDb () {
+	const validatorsCol = db.getMongo().collection('validators')
+	return validatorsCol.find({})
+		.toArray()
+		.then((res) => res)
```

Comment:
There is no need for `.then((res) => res)`, `toArray()` returns a promise that would be resolved using `await retrieveChannelsFromDb()`